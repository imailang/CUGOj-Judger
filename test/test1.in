5
1.54 2.001
546.2 64.212
120.5153 12161.51
156.1 15.1
5.5 6.6
