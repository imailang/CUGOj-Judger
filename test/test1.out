3.541
610.412
12282.0253
171.2
12.1